docs/examples/ringach97.m

```diff
@@ -1,23 +1,38 @@
+%% Tuning of orientation detectors in human vision
+% Ringach, DL. (1998) Vision Research 38(7): 963-72
+% https://doi.org/10.1016/S0042-6989(97)00322-2
+%
+% This script replicates the above psychophysics experiment using Signals.
+% When play is pressed the experiment begins and a sequence of gratings of
+% different orientations are presented in quick succession.  The subject
+% must press the ctrl key as quickly as possible each time a chosen
+% orientation is observed (i.e. vertical, horizontal or oblique).  As the
+% subject does this the chosen orientations are plotted as a histogram.
+% The observered distribution follows a 'Mexican hat' shape.
+
+%% Parameters
 oris = 0:18:162; % set of orientations, deg
 phases = 90:90:360; % set of phases, deg
-presentationRate = 30; % Hz
+presentationRate = 10; % Hz
 sf = 0.2; %spatial frequency, cyc/deg
 winlen = 10; % length of histogram window, frames
 
+%% Figure window
 figh = figure('Name', 'Press ctrl on horizontal grating',...
   'Position', [680 250 560 700], 'NumberTitle', 'off');
 vbox = uiextras.VBox('Parent', figh);
-[t, setElemsFun] = sig.playground([], vbox);
+[t, setElemsFun] = sig.playgroundPTB([], vbox);
 sigbox = t.Node.Net;
 axh = axes('Parent', vbox, 'NextPlot', 'replacechildren', 'XTick', oris);
 xlabel(axh, 'Orientation');
 ylabel(axh, 'Time (frames)');
 ylim([0 winlen] + 0.5);
+
 %% Signals stuff
 % Create a signal of WindowKeyPressFcn events from the figure
 keyPresses = sigbox.fromUIEvent(figh, 'WindowKeyPressFcn'); 
 % Create a filtered version, only keeping Ctrl presses. Turn each into 'true'
-reports = keyPresses.keepWhen(strcmp(keyPresses.Key, 'control')).map(true);
+reports = keyPresses.keepWhen(strcmp(keyPresses.Key, 'space')).map(true);
```

Comment:
Shall we change this back to control?  I worry the play/pause button focus will annoy or confuse people.